ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/ip-nat.md

```diff
@@ -0,0 +1,102 @@
+# ip-nat
+
+## Table of Contents
+
+- [Management](#management)
+  - [Management Interfaces](#management-interfaces)
+- [IP NAT](#ip-nat)
+  - [NAT Pools](#nat-pools)
+  - [NAT Synchronization](#nat-synchronization)
+  - [NAT Translation Settings](#nat-translation-settings)
+
+## Management
+
+### Management Interfaces
+
+#### Management Interfaces Summary
+
+##### IPv4
+
+| Management Interface | description | Type | VRF | IP Address | Gateway |
+| -------------------- | ----------- | ---- | --- | ---------- | ------- |
+| Management1 | oob_management | oob | MGMT | 10.73.255.122/24 | 10.73.255.2 |
+
+##### IPv6
+
+| Management Interface | description | Type | VRF | IPv6 Address | IPv6 Gateway |
+| -------------------- | ----------- | ---- | --- | ------------ | ------------ |
+| Management1 | oob_management | oob | MGMT | - | - |
+
+#### Management Interfaces Device Configuration
+
+```eos
+!
+interface Management1
+   description oob_management
+   vrf MGMT
+   ip address 10.73.255.122/24
+```
+
+## IP NAT
+Kernel Buffer Size: 64 MB
+
+### NAT Pools
+- Pool Name: **prefix_16**
```

Comment:
Try to follow the style of router_bgp.peer_groups for creating sections for each nat pool. Please use a table for the settings instead of these lines above.